stopstalk_app/lib/widgets/app_drawer.dart

```diff
@@ -141,23 +141,6 @@ class _AppDrawerState extends State<AppDrawer> {
               },
             ),
             Divider(),
-            ListTile(
-              leading: Icon(Icons.sort),
-              title: Text('Submission Filters'),
-              onTap: () {
-                Navigator.of(context)
-                    .pushReplacementNamed(SubmissionFiltersScreen.routeName);
-              },
-            ),
-            Divider(),
-            ListTile(
-              leading: Icon(Icons.favorite),
-              title: Text('Testimonials'),
```

Comment:
@gaditi123 are we not keeping this?

Reply:
According to the list Raj had sent, these topics aren't important for the first release. So removed them for now.